src/Service/VrijBrpService.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace CommonGateway\VrijBRPToZGWBundle\Service;
+
+class VrijBrpService
+{
+
+
+    public function setVrijBRPDefaults(array $configuration): array
```

Comment:
Missing doc comment for function setVrijBRPDefaults()

Reply:
Fixed in PR 4